src/Drawable.js

```diff
@@ -585,6 +612,28 @@ class Drawable {
         id |= (b & 255) << 16;
         return id + RenderConstants.ID_NONE;
     }
+
+    /**
+     * Sample a color from a drawable's texture.
+     * @param {twgl.v3} vec The scratch space [x,y] vector
+     * @param {Drawable} drawable The drawable to sample the texture from
+     * @param {Uint8ClampedArray} dst The "color4b" representation of the texture at point.
+     * @returns {Uint8ClampedArray} The dst object filled with the color4b
+     */
+    static sampleColor4b (vec, drawable, dst) {
+        const localPosition = getLocalPosition(drawable, vec);
+        if (localPosition[0] < 0 || localPosition[1] < 0 ||
+            localPosition[0] > 1 || localPosition[1] > 1) {
+            dst[3] = 0;
+            return dst;
+        }
+        const textColor =
+        // commenting out to only use nearest for now
+        // drawable.useNearest ?
+             drawable.skin._silhouette.colorAtNearest(localPosition, dst);
+        // : drawable.skin._silhouette.colorAtLinear(localPosition, dst);
```

Comment:
Could you file an issue to track why `colorAtLinear` is disabled / what we need to do if we want to enable it / why we should or shouldn't do so?

Reply:
so it's like 5* the math to sample linear (and i think the math might actually be buggy for alpha pixels) - that's mostly why